gammapy/makers/safe.py

```diff
@@ -220,38 +247,54 @@ def make_mask_energy_edisp_bias(self, dataset, observation=None):
             Safe data range mask.
         """
         edisp, geom = dataset.edisp, dataset._geom
-        position = None
 
-        if self.fixed_offset is not None:
-            if observation:
-                pointing = observation.get_pointing_icrs(observation.tmid)
-                position = pointing.directional_offset_by(
-                    position_angle=0 * u.deg, separation=self.fixed_offset
-                )
-            else:
-                raise ValueError(
-                    f"{observation} argument is mandatory with {self.fixed_offset}"
-                )
+        if self.irfs == "DL3":
+            offset = self.fixed_offset
+            if offset is None:
+                if self.position:
+                    offset = observation.get_pointing_icrs(observation.tmid).separation(
+                        self.position
+                    )
+                else:
+                    offset = 0.0 * u.deg
+
+            edisp = observation.edisp.to_edisp_kernel(offset=offset)
+            energy_min = edisp.get_bias_energy(self.bias_percent / 100)[-1]
```

Comment:
```suggestion
            energy_min = edisp.get_bias_energy(self.bias_percent / 100)[0]
```
as for the DL4 case ?